Makefile

```
@@ -36,6 +37,18 @@ format_check:
 	$(FORMATTER) $(CHECK_DIR) --check --diff
 	$(IMPORT_SORTER) $(CHECK_DIR) --check --diff
 
+.PHONY: cov
+cov:
+	$(PYTEST) $(COVERAGE_OPT) --cov-report html $(TEST_DIR)
```

python-project-templateでは`CHECK_DIR`が適切そうですね

```suggestion
	$(PYTEST) $(COVERAGE_OPT) --cov-report html $(CHECK_DIR)
```